pyproject.toml

```diff
@@ -31,6 +31,8 @@ dependencies = [
     'fonticon-materialdesignicons6',
     'qtpy',
     'tifffile',
+    'PyYAML',
+    'sympy'
```

Comment:
I haven't looked closely... but I wonder if this dependency is really necessary.  Let's discuss

Reply:
(in general, it's best to avoid adding dependencies if at all possible)